woodwork/statistics_utils/_calculate_max_dependence_for_pair.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import pandas as pd
+
+
+def _calculate_max_dependence_for_pair(
+    result, returned_measures, min_shared, extra_stats
+):
+    """Calculates the max dependence measure for a pair of columns. While
+    the magnitude of a negatively correlated value is used to determine the max,
+    the original (negative) value will be used in the "max" column.
+    """
+    # if pearson was not measured, mutual info must be max
+    if "pearson" in result:
+        score = pd.Series(
+            [result["mutual_info"], abs(result["pearson"])],
```

Comment:
Since we are determining the max, we are in a scenario where both mutual and Pearson were calculated.  Since the columns we can use for Pearson are a subset of the columns we can use for Mutual Info, we'll always have a mutual info score but might not have a Pearson score.